### PR TITLE
AutoTestgridGen: Use ghproxy and mark as rehearsable

### PR DESCRIPTION
**Pull request by GeoBK**

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -1260,12 +1260,13 @@ periodics:
...
   labels:
     ci.openshift.io/role: infra
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-prow-auto-testgrid-generator
   spec:
     containers:
     - args:
       - --github-token-path=/etc/github/oauth
-      - --github-endpoint=https://api.github.com
+      - --github-endpoint=http://ghproxy
```
Comment (petr-muller) [inline ci-operator/jobs/infra-periodics.yaml:1269]: Looks like our other jobs keep the direct URL as a backstop when the proxy is down, so we should do it here, too:

```
       - --github-endpoint=http://ghproxy                                                                                                                                                 
       - --github-endpoint=https://api.github.com
```

**Review by petr-muller — COMMENTED** [review]
You'll need to move the job to the appropriate part https://github.com/openshift/release/blob/master/core-services/sanitize-prow-jobs/_config.yaml to move it between clusters. It also looks like the ncessary secret is not on app.ci  (`The pod could not start because it could not mount the volume "token": secret "github-credentials-openshift-bot" not found`)

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (GeoBK) [reply ci-operator/jobs/infra-periodics.yaml:1269]: Will just adding this work, ie won't I need to make changes in the code to support this backstop behavior?

**Comment by GeoBK** [comment]
> You'll need to move the job to the appropriate part https://github.com/openshift/release/blob/master/core-services/sanitize-prow-jobs/_config.yaml to move it between clusters. It also looks like the ncessary secret is not on app.ci (`The pod could not start because it could not mount the volume "token": secret "github-credentials-openshift-bot" not found`)

@petr-muller To add the secret "github-credentials-openshift-bot" to app.ci, I would have to add make the required change in secret-bootstrap config file and then run the tool right?

File: ci-operator/jobs/infra-periodics.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply ci-operator/jobs/infra-periodics.yaml:1266]: No, that is the behaviour already with multiple entries

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**